### DISCUSSION TITLE
Self-hosted runner changelog

### DISCUSSION THREAD

**Discussion by jnewland** [community/community#25507] [Actions]
<p>I noticed that the version of the self-hosted actions runner changed recently and wondered what had changed. I understand that the runner <a href="https://github.community/t/self-hosted-runner-source-code/16937">isn’t open source yet</a>. Until then, is there a changelog available? Thanks!</p>

**Comment by dakale** [comment, marked as answer]
<p>Good timing, we actually just open sourced the runner today: <a href="https://github.com/actions/runner/releases" rel="nofollow noopener">https://github.com/actions/runner/releases</a> </p>
